### DISCUSSION TITLE
Sonar Error Build

### DISCUSSION THREAD

**Discussion by yJeffDev** [community/community#54267] [Actions] [answered]
### Select Topic Area

Question

### Body

My pipeline error:
Error:  Failed to execute goal org.sonarsource.scanner.maven:sonar-maven-plugin:3.9.0.2155:sonar (default-cli)

What I do in this case?

**Comment by lucasscaramelo** [comment, marked as answer]
I already saw this error once and found the answer in a debate here: https://github.com/AxonFramework/extension-kafka/issues/194

**Comment by yJeffDev** [comment]
Thanks!
